includes/class-wc-siftscience-admin.php

```diff
@@ -249,93 +262,119 @@ public function save_settings() {
+					'Scores below this value are considered good and shown in green',
+					array( 'default' => 30 )
+				),
 
-				$this->get_text_input( WC_SiftScience_Options::NAME_PREFIX,
+				$this->get_element(
+					'number',
+					WC_SiftScience_Options::THRESHOLD_BAD,
+					'Bad Score Threshold',
+					'Scores above this value are considered bad and shown in red',
+					array( 'default' => 60 )
+				),
+				$this->get_element(
+					'text',
+					WC_SiftScience_Options::NAME_PREFIX,
 					'User & Order Name Prefix',
-					'Prefix to give order and user names. '
-					. 'Useful when you have have multiple stores and one Sift account.' ),
-
-				$this->get_check_box( WC_SiftScience_Options::AUTO_SEND_ENABLED,
-					'Automatically Send Data',
-					'Automatically send data to Sift when an order is created'
+					'Prefix to give order and user names. Useful when you have have multiple stores and one Sift account.'
 				),
 
-				$this->get_number_input( WC_SiftScience_Options::MIN_ORDER_VALUE,
+				$this->get_element( 'checkbox', WC_SiftScience_Options::AUTO_SEND_ENABLED, 'Automatically Send Data', 'Automatically send data to Sift when an order is created' ),
+
+				$this->get_element(
+					'number',
+					WC_SiftScience_Options::MIN_ORDER_VALUE,
 					'Minimum Order Value for Auto Send',
-					'Orders less than this value will not be automatically sent to sift. Set to zero to send all orders.', 0 ),
+					'Orders less than this value will not be automatically sent to sift. Set to zero to send all orders.',
+					array( 'default' => 0 )
+				),
 
-				$this->get_section_end( 'sifsci_section_main' ),
+				$this->get_element( 'sectionend', 'sifsci_section_main' ),
 			);
 		}
-
-		private function get_title( $id, $title ) {
-			return array( 'title' => $title, 'type' => 'title', 'desc' => '', 'id' => $id );
-		}
-
-		private function get_text_input( $id, $title, $desc ) {
-			return array(
-				'title' => $title,
-				'desc' => $desc,
-				'desc_tip' => true,
-				'type' => 'text',
-				'id' => $id,
+		/**
+		 * This function sets HTML element attributes according to woocommearce provided library.
+		 *
+		 * @param String $type            Element type name.
+		 * @param String $id              HtmlElement ID.
+		 * @param String $title           Element label.
+		 * @param String $desc            Question mark hilper title.
+		 * @param Array  $element_options Element special options.
+		 *
+		 * @return Array $element         An array of attributes.
+		 */
+		private function get_element( $type, $id, $title = '', $desc = '', $element_options = array() ) {
+
+			$element = array(
+				'type' => $type,
+				'id'   => $id,
 			);
-		}
 
-		private function get_number_input( $id, $title, $desc, $default ) {
-			return array(
-				'id' 		=> $id,
-				'title' 	=> $title,
-				'desc' 		=> $desc,
-				'default' 	=> $default,
-				'type' 		=> 'number',
-				'desc_tip' 	=> true
-			);
-		}
+			switch ( $type ) {
+				case 'sectionend':
+					return $element;
```

Comment:
My personal preference is that a switch statement either always use "return" or always use "break". Otherwise, things get confusing.

Reply:
no comment